cfme/base/ui.py

```diff
@@ -412,6 +416,18 @@ class DiagnosticsCollectLogsEdit(DiagnosticsCollectLogsView):
     def is_displayed(self):
         return super(DiagnosticsCollectLogsEdit, self).is_displayed and self.protocol.is_displayed
 
+    type = BootstrapSelect('log_protocol')
```

Comment:
Here you're overloading the builtin 'type', IIRC there were comments on @izapolsk PR about doing this, and it was determined to be admissible in these cases because the widget is referenced through the view.

Consider changing it to better describe the field and avoid the potential collision. `log_type` maybe?